app/controllers/downvotes_controller.rb

```diff
@@ -7,30 +7,34 @@ def create
         review_id: @review.id,
         user_id: current_user.id
       )
-      @review.score -= 1
-
-      if @downvote.save && @review.save
-        flash[:notice] = "Downvote created successfully."
-        redirect_to @beach
-      else
-        flash[:notice] = "Downvote not created successfully."
-        render :new
+      respond_to do |format|
+        if @downvote.save
+          format.html { redirect_to :back,
+            notice: "Downvote created successfully." }
+          format.json { render json: @downvote }
+        else
+          flash[:notice] = "Downvote not created successfully."
+          format.html { redirect_to :back }
+          format.json { render json: @downvote.errors,
+            status: :unprocessable_entity }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 19, 43 should be on its own line.